crates/pipeline-manager/src/db/operations/pipeline.rs

```diff
@@ -1601,15 +1601,16 @@ pub(crate) async fn get_next_rust_compilation(
 }
 
 /// Retrieves the list of successfully compiled pipeline programs (pipeline identifier, program version,
-/// program binary source checksum, program binary integrity checksum) across all tenants.
+/// program binary source checksum, program binary integrity checksum) AND pipeline programs that
```

Comment:
comment above is outdated (;..of successfully...)

is theGC the only place where we call this function? otehrwise it changes the semantics in other places

Reply:
`Retrieves the list of successfully compiled pipeline`

Reply:
line 1603

Reply:
on next line we have `AND pipeline programs that are currently being compiled` the thing in (..) is just the things it would return